test/_unit/behaviours/get-section.spec.js

```diff
@@ -0,0 +1,314 @@
+'use strict';
+
+const Controller = require('hof').controller;
+const Behaviour = require('../../../apps/paf/behaviours/get-section');
+
+describe('apps/paf/behaviours/get-section', () => {
+  describe('configure', () => {
+    let controller;
+    let req;
+    let res;
+    let sandbox;
+
+    beforeEach(done => {
+
+      req = reqres.req();
+      res = reqres.res();
+      req.sessionModel.attributes.steps = [];
+
+      const GetSectionController = Behaviour(Controller);
+      controller = new GetSectionController({ template: 'index', route: '/index' });
+      controller._configure(req, res, done);
+    });
+
+
+    describe(`sets the 'next' property for pages preceeding the first page of each section`, () => {
+      beforeEach(() => {
+        sandbox = sinon.createSandbox();
+        req.form.options = {
+          steps: {
+            '/crime-location': {},
+            '/crime-another-location': {},
+            '/report-person': {},
+            '/has-additionalPerson': {},
+            '/person-details': {},
+            '/report-organisation': {},
+            '/another-company': {},
+            '/other-info-file-upload': {},
+            '/about-you-contact': {},
+            '/are-you-eighteen': {},
+          }
+        };
+
+        sandbox.stub(Controller.prototype, 'configure').yieldsAsync();
+      });
+
+      afterEach(() => {
+        sandbox.restore();
+      });
+
+      it(`the crime-location page has the 'next' property as '/report-person'`, () => {
```

Comment:
this is good maybe write it as `next page` and below

Reply:
thanks Sulthan, I have amended those as suggested